gui-agent/vmside.c

```diff
@@ -1041,21 +1044,31 @@ void process_xevent_selection_req(Ghandles * g,
                 tmp, sizeof(tmp) / sizeof(tmp[0]));
         resp.property = req->property;
     }
-    if (req->target == Utf8_string_atom)
-        convert_style = XUTF8StringStyle;
     if (req->target == XA_STRING)
         convert_style = XTextStyle;
-    if (req->target == Compound_text)
+    else if (req->target == Compound_text)
         convert_style = XCompoundTextStyle;
+    else if (req->target == Utf8_string_atom)
+        convert_style = XUTF8StringStyle;
     if (convert_style != XConverterNotFound) {
         XTextProperty ct;
-        Xutf8TextListToTextProperty(g->display,
-                (char **) &g->clipboard_data,
-                1, convert_style, &ct);
+        char **ptr = { NULL };
```

Comment:
This should be `char *ptr[0]`, otherwise it points at an initializer in rodata section.

And indeed, when pasting characters from the very issue title it tries to fix, I got SEGV.

We do have some tests for inter-VM clipboard (https://github.com/QubesOS/qubes-core-admin/blob/master/qubes/tests/integ/basic.py#L468), maybe we need one for weird unicode characters too? (but that isn't blocking for this PR)